<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createNewPreset">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QListView" name="viewPresets">
    <property name="geometry">
     <rect>
      <x>705</x>
      <y>161</y>
      <width>161</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="resizeMode">
     <enum>QListView::Fixed</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSetup">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Setups</string>
    </property>
    <widget class="QPushButton" name="runButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="editButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>290</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="importButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Import</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>320</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="quickSettingsButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quick Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
